app/__init__.py

```diff
@@ -35,11 +47,12 @@ def create_app(config_name):
         if not framework_data['slug'] in application.config.get('DM_FRAMEWORK_CONTENT_MAP', {}):
             if framework_data['framework'] == 'g-cloud':
                 if framework_data['status'] != 'expired':
-                    content_loader.load_manifest(framework_data['slug'], 'services', 'search_filters')
```

Comment:
I'm not sure if we should go the approach of hiding errors by default. Instead can we build further on the statuses we expect a manifest to exist for? e.g. not for open or expired frameworks (or whatever it is). It feels better to be explicit about what state we want need our manifests to be in for an app to start rather than assume it is set up in that way). Saying that, I had assumed that unit tests would have picked up on this rather than us finding out in the way we did which wasn't ideal. 

If we did decide to use the try_load then we could also remove the framework status checks as they are essentially redundant.

Feel free to argue the counter these points, my opinions aren't too strongly held.

Reply:
Also just to double check, for the occasions we are using try load instead of just loading, must we do so every time (e.g. can be more explicit sometimes and be less others based on framework type/state)

Reply:
So I don't think it's possible to define by framework status what we'd expect to be there.  I guess the thing we're avoiding by doing the `try` for everything is having to update content in all of the frontend repos before you can add a new framework even only on Preview.

Maybe that's being  lazy and we should have the new framework everywhere, I don't know.  But feels to me like for applications to open we should only need buyer and supplier apps to have the content, as they are the only apps that do anything to a framework before applications are closed.

Reply:
I don't think building out the statuses bit here is necessarily a good idea (and if we were to go that route, that's something that would need to start to be centralised, probably). Try/load takes us a step towards consistency (Eg with the admin-frontend) and limits the damage the problem can cause in the future. If we want to take a closer look at this, it's probably something we should do as tech debt/scalability issue.

Reply:
Cool, sounds like everyone else is also behind that approach so I'll follow your lead 👍